packages/admin-portal/src/reducers/apps/list.ts

```diff
@@ -0,0 +1,56 @@
+import { Action } from '@/types/core'
+import { isType } from '@/utils/actions'
+import { appsReceiveData, appsRequestData, appsRequestFailure } from '@/actions/apps-management'
+import { PagedResultAppSummaryModel_ } from '@reapit/foundations-ts-definitions'
+import { FetchDetailResult } from '@reapit/utils'
+
+export type AppListState = PagedResultAppSummaryModel_ & Pick<FetchDetailResult<any>, 'isLoading' | 'errorMessage'>
+
+export const defaultState: AppListState = {
+  isLoading: false,
+  errorMessage: '',
+  data: [],
+  totalCount: 0,
+  pageSize: 0,
+  pageNumber: 0,
+  pageCount: 0,
+}
+
+const appListReducer = (state: AppListState = defaultState, action: Action<any>): AppListState => {
+  // Test TBC
```

Comment:
Can you include the tests in this PR please

Reply:
@willmcvay tests added